docs/examples/basic.dto.neon

```diff
@@ -7,6 +7,7 @@ Car:
 		attributes: 'string[]'
 		manufactured: '\Cake\I18n\FrozenDate'
 		owner: Owner
+		service: 'string[]|int|float'
```

Comment:
I would rather make this a test file in /tests/test_files
not the best example use case for the examples dtos IMO : )

Reply:
Gotcha, will do